lib/core/constants/quick_links.dart

```diff
@@ -1,37 +1,74 @@
-import 'package:notredame/core/models/quick_link.dart';
+// FLUTTER / DART / THIRD-PARTIES
+import 'package:flutter/widgets.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
 
+// MODELS
+import 'package:notredame/core/models/quick_link.dart';
+
+// UTILS
+import 'package:notredame/ui/utils/app_theme.dart';
+
 List<QuickLink> quickLinks(AppIntl intl) => [
       QuickLink(
           name: intl.ets_security_title,
-          image: 'assets/images/ic_security_red.png',
+          image: const FaIcon(
+            FontAwesomeIcons.shieldAlt,
+            color: AppTheme.etsLightRed,
+            size: 64,
+          ),
           link: 'security'),
       QuickLink(
           name: intl.ets_monets_title,
-          image: 'assets/images/ic_monets_sans_nom_red.png',
+          image: Image.asset(
+            'assets/images/ic_monets_sans_nom_red.png',
+            color: AppTheme.etsLightRed,
+          ),
           link: 'https://portail.etsmtl.ca'),
       QuickLink(
           name: intl.ets_bibliotech_title,
-          image: 'assets/images/ic_book_red.png',
+          image: const FaIcon(
+            FontAwesomeIcons.book,
```

Comment:
Just so you know, these icons are also inside Material Icons and so `Icons.<YOUR-ICON>` and you can change their size and colour 😄

Reply:
Yeah, I know this one is also available in material but some (Github...) aren't.